src/main/resources/com/sumologic/sumobot/http_frontend/style.css

```diff
@@ -0,0 +1,139 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+ html, body {
+    height: 100%;
+
+    padding: 0;
+    margin: 0;
+
+    background-color: #f7f7f7;
+ }
+
+ body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    font-family: 'Open Sans', sans-serif;
+ }
+
+ .page-container {
+    padding: 10px;
+
+    background-color: white;
+
+    box-shadow: 0px 0px 60px #d4d4d4;
+
+    border-radius: 20px;
+ }
+
+ #messages {
```

Comment:
It's better to not style by IDs as that has extremely high CSS specificity & it's hard to override. See https://specificity.keegan.st/. Use classes.